test/test_mobile_optimizer.py

```diff
@@ -248,8 +246,86 @@ def forward(self, x):
         model = torch.jit.script(model)
         # this line should not have ASAN failures
         model_optim = optimize_for_mobile(model)
-        self.assertFalse(hasattr(model_optim.conv1, "bias"))
-        self.assertFalse(hasattr(model_optim.child.conv2, "bias"))
+
```

Comment:
Is the understanding that one cannot serialize/deserialize a model after optimize_for_mobile is called?